Prox/Prox/Utilities/PlaceUtilities.swift

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+struct PlaceUtilities {
+    static func sort(places: [Place], byDistanceFromLocation location: CLLocation, ascending: Bool = true) -> [Place] {
+        return places.sorted { (placeA, placeB) -> Bool in
+            let placeADistance = location.distance(from: CLLocation(latitude: placeA.latLong.latitude, longitude: placeA.latLong.longitude))
+            let placeBDistance = location.distance(from: CLLocation(latitude: placeB.latLong.latitude, longitude: placeB.latLong.longitude))
+
+            if ascending {
+                return placeADistance < placeBDistance
+            }
+
+            return placeBDistance < placeADistance
```

Comment:
nit: I'd prefer the variables to keep the same order and switch the operator (`placeADistance > placeBDistance`). It's a little easier to scan.